static/app/components/smartSearchBar/index.tsx

```diff
@@ -751,7 +755,13 @@ class SmartSearchBar extends React.Component<Props, State> {
     }
 
     return [
-      tagKeys.map(value => ({value, desc: value})),
+      tagKeys
+        .map(value => ({
+          value,
+          desc: value,
+          documentation: getFieldDoc ? getFieldDoc(value.slice(0, -1)) : '',
```

Comment:
```suggestion
          documentation: getFieldDoc?.(value) ?? '',
```
I think this should be valid too, if `getFieldDoc` is undefined it won't call it, and I don't think we need to copy the string since it's a primitive

Reply:
whoa fancy, didn't realize i could do this :+1:

Reply:
oh actually this isn't copying the string, its stripping the last character cause the keys actually come as
`count():` (ie. with the extra `:` at the end)